### ISSUE TITLE
Get the ingest internal integration tests running again

### ISSUE THREAD

**Issue by justincc** [ebi-ait/hca-ebi-dev-team#24]
**Dependencies**
* Upload service is up
* Gitlab instance available

**Acceptance criteria**
* [x] Integration test runs are set up in Gitlab.
* [x] The tests for the datastore and other components beyond ingest and upload are disabled.

**Comment by aaclan-ebi** [comment]
The GitLab project for integration tests should now have access to the GitLab runner in EC2 instance and attached IAM Role to access EBI Ingest AWS resources. Documenting the steps here: https://docs.google.com/document/d/1_aeMhnqX5hb6hwZeutTY_moBBKrMkpIoCAETJIemc7M/edit#heading=h.amszcuqov05k

The Gitlab pipelines should now run up to the uploading of files. Tests will fail once it reach that point. All our tests currently involves uploading of files.

Marking ticket as blocked by the Upload service setup.

**Comment by justincc** [comment]
I would be happy to mark this as done since the tests are running even if they're failing. Once we resolve the upload issue, if tests fail for some other reason not due to missing components we could address that under another task. What do you think?

**Comment by aaclan-ebi** [comment]
Sure, acceptance criteria are met anyways.
